arrow-array/src/array/run_array.rs

```diff
@@ -251,6 +247,47 @@ impl<R: RunEndIndexType> RunArray<R> {
             values: self.values.clone(),
         }
     }
+    /// Expands the REE array to its logical form
```

Comment:
```suggestion
    }
    
    /// Expands the REE array to its logical form
```
All the other functions in this file seem to be leaving a space between them. I would try to be consistent with the previous formatting of the file